app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,28 @@
+footer {
+  margin-top: 45px;
+  padding-top: 5px;
+  border-top: 1px solid #eaeaea;
+  color: #999;
+}
+
+footer a {
+  color: #555;
+}
+
+footer a:hover {
```

Comment:
I think proper .scss style is to nest the nested attributes instead of listing them underneath each other.

Ex. `footer a { color: #555; 
&:hover { color: #222; }
}`

Same for the nested ul styling.